src/plugins/intel_cpu/src/ngraph_transformations/convert_logsoftmax.hpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <ngraph/pass/graph_rewrite.hpp>
+
+namespace ov {
+namespace intel_cpu {
+class ConvertLogSoftmax: public ngraph::pass::MatcherPass {
```

Comment:
Minor: can we use ov::pass::MatcherPass as a base class here?

Reply:
Sure no problem